README.md

```diff
@@ -1,38 +1,98 @@
 
 > Kevin K.H. Cheung, Ambros Gleixner, and Daniel E. Steffy: [Verifying Integer Programming Results](http://dx.doi.org/10.1007/978-3-319-59250-3_13). In: F. Eisenbrand and J. Koenemann, eds., Integer Programming and Combinatorial Optimization: 19th International Conference, IPCO 2017, pp. 148-160, 2017, [`doi:10.1007/978-3-319-59250-3_13`](http://dx.doi.org/10.1007/978-3-319-59250-3_13).
 
-This repository contains a detailed technical [specification of the certificate file format](http://rawgit.com/ambros-gleixner/VIPR/master/cert_spec_v1_0.html), [software](code/) to check, display, and compress certificate files, and [supplementary information](experiments/) on the computational experiments conducted for the article above.
+This repository contains a detailed technical specification of the certificate file format in [Version 1.0](cert_spec_v1_0.html) and [Version 1.1](cert_spec_v1_1.html), [software](code/) to check, display, compress, and complete certificate files, and [supplementary information](experiments/) on the computational experiments conducted for the article above.
 
+The specification and handling of incomplete derivations in [Version 1.1](cert_spec_v1_1.html) was added for the purpose of the paper
 
-## Software
+> TODO add arxiv link to preprint
+
+Please cite both publications if you use VIPR in your work.
 
-*VIPR* currently comes with three C++ programs:
-- [`viprchk`](code/viprchk.cpp): A program that verifies mixed-integer linear programming certificate files specified in the `.vipr` file format.
-- [`viprttn`](code/viprchk.cpp): A program that tightens and improves .vipr files, potentially reducing their size and allowing for easier checking.
-- [`vipr2html`](code/viprchk.cpp): A program that converts `.vipr` certificate files to a human readable HTML format (not recommended for large files).
+## Software
 
-The code should compile with most modern compilers supporting the C++11 standard.  Directory [`code`](code/) provides a basic [`makefile`](code/makefile).  We have successfully built the tools with GNU g++ version 5.4.0 under Ubuntu 16.04.4.  It depends on the [GNU Multiple Precision library](https://gmplib.org/).
+*VIPR* currently provides five C++ scripts, each being called from a terminal together with an appropriate `.vipr` certificate file:
 
+- `vprchck`: A program that verifies mixed-integer linear programming certificate files specified in the `.vipr` file format.
+- `vipr2html`: A program that converts `.vipr` certificate files to a human readable HTML format (not recommended for large files).
+- `viprttn`: A program that tightens and improves `.vipr` files, potentially reducing their size and allowing for easier checking.
+- `viprwidth`: A program that returns the cutwidth on derived constraints.
```

Comment:
- file missing?
- describe what is meant by cutwidth here

Reply:
ah, that file is actually something that Kevin wrote years ago but was never meant to be published I guess? I will remove it from the README

Reply:
Okay, when that is done I can also test the build process.